### PR TITLE
Add Console Client Resources link

### PR DESCRIPTION
**Pull request by mareklibra**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/111

### CODE REVIEW

**Comment by mareklibra** [comment]
ccr

**Comment by mareklibra** [comment]
@jniederm , what do you think from design perspective?
- The `Console Client Resources` text&link
- design allignment of `Console Options` and `List of Disks`

File: src/components/ConsoleOptions/index.js [hunk 1]
```diff
@@ -62,6 +62,9 @@ class ConsoleOptions extends React.Component {
             />
           </dd>
 
+          <div className={style['console-client-resources']}>
+            Refer <a href='https://www.ovirt.org/documentation/admin-guide/virt/console-client-resources/' target='_blank'>Console Client Resources</a> in case of troubles.
```
Comment (jelkosz) [inline src/components/ConsoleOptions/index.js]: "in case of troubles" does not sound too good to me. Maybe:
For details refer to Client Resources?

**Comment by mareklibra** [comment]
The issue is being fixed in https://github.com/oVirt/ovirt-web-ui/pull/221/commits/6b2a17dcb6a5f05a102be59d73b94f530187b273 - "console client resources" link is rendered within overlay-help

File: src/components/ConsoleOptions/index.js (hunk 1 above)
Comment (jniederm) [inline src/components/ConsoleOptions/index.js]: Tested on version: commit 252b932, components commit c00b232

* Console block is nicely aligned in contrast to disks: I find disks unnecessarily shifted to the right and a bit illogically starting at the same line as the label.
* When "hide" link is clicked in Console section then the hide-able block jumps left.
* I'd suggest to use arrows down and right in accordance with [1].

[1]: https://www.patternfly.org/pattern-library/forms-and-controls/expand-collapse-section/#/overview

**Pull request closed without merging**